docs/developer/app-store/apps/cms.mdx

```diff
@@ -160,6 +161,32 @@ Each of the CMS platforms has slight differences in configuration and behavior d
 - Fields are not autocompleted. You must ensure mapping is correct.
 - Builder.io doesn't support unique constraints on fields - hence the app will not guarantee that duplicates will not be created.
 
+### [Payload](https://payloadcms.com/)
+
+Payload CMS requires some extra steps to be performed before the app can be used.
+
+The app uses REST API, by default enabled in Payload. Typically, it's available under `/api` path.
+You need to put this URL in the configuration form.
```

Comment:
**issue**:
Those two sentences aren't super clear to me.

> The app uses REST API

To me, that suggests that the app has its own REST API. I think you meant that the app communicates with Payload's REST API.

> Typically, it's available under `/api` path.

We could use some more context here. Maybe a link to the docs?

> You need to put this URL in the configuration form.

Maybe we can name a specific field in which we need to put the URL.